composer/core/state.py

```diff
@@ -73,9 +73,10 @@ class State(Serializable):
         precision (str | Precision): The numerical precision to use for training. See :class:`~.Precision` for
             the supported precisions.
         precision_context (Callable[[Precision], ContextManager]): Function to produce a context manager to mandate precision.
-        optimizers (:attr:`~.types.Optimizers`, optional): The optimizers being used to train the model. Multiple optimizers are not currently supported.
-        schedulers (:attr:`~.types.PyTorchScheduler` | List[:attr:`~.types.PyTorchScheduler`] | Tuple[:attr:`~.types.PyTorchScheduler`, ...], optional):
-            The learning rate scheduler (can also be a list or tuple of schedulers).
+        optimizers (:attr:`~.types.Optimizers`, optional): The optimizers being used to train the model.
+            Multiple optimizers are not currently supported.
+        schedulers (Many[:attr:`~.types.PytorchScheduler`], optional): The learning rate scheduler
```

Comment:
While more concise, I would prefer not to use the `Many` notation in docstrings. Instead, could we do something like this?
```suggestion
        schedulers (types.PyTorchScheduler | Sequence[types.PyTorchScheduler], optional): The learning rate scheduler
```
```suggestion
        schedulers (Many[:attr:`~.types.PytorchScheduler`], optional): The learning rate scheduler
```

Reply:
Unfortunately the new Union Operator `|` is only in Python 3.10 (PEP 604).

Reply:
Yup, the bitwise or operator is fine in docstrings (nothing is really checking it anyways) + more concise and clear than `Union[...]`